src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -21,16 +30,43 @@ export const BaseCourseSearchResults = ({
       accessor: 'advertised_course_run.key',
     },
   ];
+
   const initialState = useMemo(() => ({
     pageSize: searchResults?.hitsPerPage,
-    pageIndex: searchResults?.pageIndex || 0,
-    pageCount: searchResults?.nbPages || 1,
-  }), [searchResults?.pageIndex]);
+    pageIndex: searchResults?.page || 0,
+  }), [searchResults?.page, searchResults?.hitsPerPage]);
 
   const [modalOpen, setModalOpen] = useState(false);
   const [selectedCourseRuns, setSelectedCourseRuns] = useState([]);
   const [showToast, setShowToast] = useState(false);
+  const fetchData = (newData) => {
```

Comment:
It might. It's the name of the method on the DataTable component, so... yeah. I'm debating changing the name on that component to something like, onChange, but for now, this is what it is. =/